10-angular/introduction_to_angular/readme.md

```diff
@@ -71,20 +71,11 @@ Files/folders:
 $ mkdir todo_angular
 $ cd todo_angular
 $ touch index.html
-$ mkdir js
-$ mkdir js/controllers
-$ touch js/app.js
+$ mkdir app
+$ mkdir app/controllers
+$ touch app/app.js
 ```
```

Comment:
This is a different file structure than the [Grumblr app](https://github.com/ga-dc/grumblr_angular/tree/0ac31e508d4f9bb1313837bd3376f9eb0faee257). Not necessarily a bad thing, but if we're looking to be consistent throughout lessons we should settle on something.